### PR TITLE
Shows a prompt when Bluetooth is off and trying to add a device

### PR DESCRIPTION
**Pull request by legendgroupv2**
When trying to add a Meshtastic device, the app will check if Bluetooth is turned on, if it isn't then it will display a message.

### CODE REVIEW

File: app/src/main/java/com/geeksville/mesh/ui/SettingsFragment.kt [hunk 1]
```diff
@@ -501,6 +503,18 @@ class SettingsFragment : ScreenFragment("Settings"), Logging {
         }
     }
 
+    private fun checkBTEnabled(
+        warningReason: String = getString(R.string.requires_bluetooth)
+    ) {
+
+        var btAdapter = BluetoothAdapter.getDefaultAdapter();
+
+        if (!(btAdapter.isEnabled())) {
```
Comment (andrekir) [inline app/src/main/java/com/geeksville/mesh/ui/SettingsFragment.kt]: Bluetooth state is handled via `bluetoothViewModel.enabled.value`, then `BluetoothAdapter` import can also be removed.

```suggestion
        if (bluetoothViewModel.enabled.value == false) {
```

**Review by andrekir — CHANGES_REQUESTED** [review]
@legendgroupv2 Thanks for your contribution!
Please check my reviews 👍

**Comment by legendgroupv2** [comment]
Thanks for your reviews. All done :)

**Review by andrekir — APPROVED** [review]
thanks

**Pull request merged**